src/graphql/account.rs

```diff
@@ -418,6 +432,35 @@ impl AccountMutation {
         update_jwt_expires_in(expires_in)?;
         Ok(time)
     }
+
+    /// Updates only the user's language setting, regardless of the user's role.
```

Comment:
"regardless of the user's role" seems unnecessary.

Reply:
I removed that part.